### DISCUSSION TITLE
BasicVSR BD在Vid4上测试只有20dB

### DISCUSSION THREAD

**Discussion by Shuyun-Wang** [open-mmlab/mmediting#1271] [Datasets] [answered]
我用BasicVSR BD的模型在Vid4上测试只有20dB，请问是哪里出现了问题呢？更改的地方如下：
![image](https://user-images.githubusercontent.com/27934055/177555685-5a5a2109-50ba-44a1-99ec-dc798f24cce3.png)
![image](https://user-images.githubusercontent.com/27934055/177555809-c953630a-4504-4487-ae2e-1cf250559a50.png)

**Comment by zengyh1900** [comment]
@Z-Fran 
Please check this issue.

**Comment by Z-Fran** [comment, marked as answer]
Sorry for the late reply. I have tested Vid4/BDx4 with the same config as you,  and PSNR is 27.98dB. Please check your data  or fetch latest code.
